docs/src/main/sphinx/connector/redshift.rst

```diff
@@ -77,13 +77,18 @@ Type mapping
 
 .. include:: jdbc-type-mapping.fragment
 
+.. _redshift-sql-support:
+
 SQL support
 -----------
 
-The following SQL statements are not yet supported:
+The connector provides read access and write access to data and metadata in
+Redshift. In addition to the :ref:`globally available
+<sql-globally-available>` and :ref:`read operation <sql-read-operations>`
+statements, the connector supports the following features:
 
-* :doc:`/sql/grant`
-* :doc:`/sql/revoke`
-* :doc:`/sql/show-grants`
+* :doc:`/sql/insert`
+* :doc:`/sql/delete`
+* :ref:`sql-schema-table-management`
```

Comment:
Turns out `sql-schema-table-management` isn't fine-grained enough for most connectors. Redshift and most JDBC connectors doesn't support ALTER SCHEMA but rest of it seems correct.

Reply:
If that's the case, then the best solution would be to include the specific statements from schema table management that are supported. I added them all (except for ALTER SCHEMA) here, if you can take a look.

If it's the same for other JDBC connectors, we ought to do the same there.

Reply:
Thanks. Looks good now. See https://github.com/trinodb/trino/pull/8667#discussion_r677574922 for the state of ALTER SCHEMA. (Only Hive and Iceberg support it AFAIK).

Reply:
Updated